core/src/autogluon/core/scheduler/seq_scheduler.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import time
+from collections import OrderedDict
+from copy import deepcopy
+
+from autogluon.core.searcher import GPFIFOSearcher
+from autogluon.core.utils import EasyDict
+from tqdm.auto import tqdm
+
+logger = logging.getLogger(__name__)
+
+
+class LocalReporter:
+    def __init__(self, trial, config, training_history: dict, config_history: dict):
+        self.trial = trial
+
+        self.training_history = training_history
+        if trial not in self.training_history:
+            self.training_history[trial] = []
+
+        self.task_config = EasyDict(deepcopy(config))
+
+        self.config_history = config_history
+
+        self.trial_started = time.time()
+        self.last_reported_time = self.trial_started
+        self.last_result = None
+
+    def __call__(self, *args, **kwargs):
+        result = deepcopy(kwargs)
+
+        if 'done' not in result:
+            result['trial'] = self.trial
+
+            now = time.time()
+            result['time_this_iter'] = now - self.last_reported_time
+            result['time_since_start'] = now - self.trial_started
+            self.last_reported_time = now
+
+            self.training_history[self.trial].append(result)
+
+            if self.trial not in self.config_history:
+                self.config_history[self.trial] = self.task_config
+                if 'util_args' in self.task_config:
```

Comment:
This is also not done in FIFOScheduler

Reply:
This property carries model objects, which might be very big. I started to removed these after serialization issues in Ray scheduler.